src/helpers/AddressHelpers.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Formats addresses as [CAIP-10](https://chainagnostic.org/CAIPs/caip-10) addresses for it's respective request scope. See [CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md) and .
+ * @param scope - The scope to create session for.
+ * @param addresses - The addresses to format. If address is empty, we remove it from the array.
+ * @returns The formatted addresses with the scope to create session for.
+ */
+export const getCreateSessionOptionalScopesFormattedAddresses = (
```

Comment:
```suggestion
export const getCaip25FormattedAddresses = (
```

Reply:
resolved in [643f141d3](https://github.com/MetaMask/test-dapp-multichain/commit/643f141d3ec91a210596cb4ec4a6a86ed5e54e29)